test/standalone/x86_cpu_model/build.zig

```diff
@@ -0,0 +1,57 @@
+const std = @import("std");
+
+pub fn build(b: *std.Build) void {
+    const test_step = b.step("test", "Test it");
+    b.default_step = test_step;
+
+    const target = b.standardTargetOptions(.{});
+    const optimize = b.standardOptimizeOption(.{});
+
+    const arch = target.result.cpu.arch;
+    if (!arch.isX86()) return;
+
+    const abi = target.result.abi;
+    if (target.result.ofmt != .elf or !(abi.isMusl() or abi.isGnu())) return;
```

Comment:
Why limit to these targets?

Reply:
Oh, I see. On macOS:

```
/Users/runner/work/zig/zig/build/../lib/std/Build/Step/CheckObject.zig:506:17: 0x10a7671e1 in checkInDynamicSymtab (build)
        else => @panic("Unsupported target platform"),
```

But really, since the test is now actually testing multiversioning at runtime, is there any value in doing these symbol checks anyway?